Source/BoundaryConditions/PEC_Insulator.cpp

```diff
@@ -142,10 +142,16 @@ namespace
                                 amrex::Real const rmirror = ijk_mirror[idim] + 0.5_rt*(1._rt - is_nodal[idim]);
                                 // Calculate radial scale factor
                                 sign *= rmirror/rguard;
+#if defined(WARPX_DIM_RSPHERE)
+                                sign *= rmirror/rguard;
+#endif
                             }
                             if (isInsulatorBoundary) {
                                 // Apply radial scale factor
                                 field_value *= dom_hi[idim]/rguard;
+#if defined(WARPX_DIM_RSPHERE)
+                                field_value *= dom_hi[idim]/rguard;
```

Comment:
Could you add a comment here, explaining why this needs to be done, for spherical geometry?

Reply:
As above, the scaling if proportional the surface area which varies as r-squared.